js/ui/scheduler/ui.scheduler.subscribes.js

```diff
@@ -243,6 +243,69 @@ const subscribes = {
         this._workSpace.restoreScrollTop();
     },
 
+    getText(data, currentData, format) {
```

Comment:
getText returns an object with data formatted for output. I suggest thinking of a more appropriate name.